src/sagas/favorites.js

```diff
@@ -0,0 +1,59 @@
+import { takeEvery, put, select } from 'redux-saga/effects';
+import favoriteActions from '../actions/favorites';
+import Utils from '../utils/Utils';
+import selectors from '../selectors';
+
+export function* getFavoritesList() {
+    const localFavs = JSON.parse(localStorage.getItem('favList')) || [];
+    yield put(favoriteActions.creators.getFavoritesListComplete(localFavs));
+}
+
+export function* addFavorite(fav) {
+    const localFavs = JSON.parse(localStorage.getItem('favList')) || [];
+    localFavs.push(fav.payload);
+    const newLocal = JSON.stringify(localFavs);
+    localStorage.setItem('favList', newLocal);
+    yield put(favoriteActions.creators.addFavoriteComplete(localFavs));
+}
+
+export function* removeFavorite(id) {
+    const localFavs = JSON.parse(localStorage.getItem('favList')) || [];
+    let tgtIndex;
+    localFavs.filter((fav, index) => {
+        if (fav.id === id.payload) {
+            tgtIndex = index;
+        }
+        return fav;
+    });
+    localFavs.splice(tgtIndex, 1);
+    const newLocal = JSON.stringify(localFavs);
```

Comment:
I think  `favsSerialized` would be a better name than `newLocal`. If you read `newLocal` without context it doesn't give you any clue of what is representing.